src/Components/Common/Button/Button.jsx

```diff
@@ -87,6 +87,7 @@ Button.defaultProps = {
   icon: null,
   loading: false,
   outline: false,
+  onClick: null,
```

Comment:
Here I think the click on a button is a must and shouldn't be optional

Reply:
This button is used in the forms without the onClick event on it. That why I didn't put it as required